examples/advanced-project/src/__checks__/synthetics/04-login.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { test } from '@playwright/test'
+
+// This check logs in as a user would, by clicking and entering text on your web page. 
+
+test('login', async ({ page }) => {
+  // See Checkly's documentation on secrets and variables: https://www.checklyhq.com/docs/browser-checks/variables/
+  const password = process.env.WEB_SHOP_PASSWORD || 'defaultPwd';
```

Comment:
I understand that that we just want to show off the env and secrets and stuff. :) How ever just making the pw configurable is a bit odd. Should the user also be an env var?